### DISCUSSION TITLE
few plots pass filter - explanation request

### DISCUSSION THREAD

**Discussion by stlanzel** [Chia-Network/chia-blockchain#5903] [Q&A]
I have currently 244 plots, but I can see only few plots (0, 1, 2 or rarely 3) passed the plot filter.
It is the same since I had 50 plots....
I expect the filter passed plots should increase proportionately to total amount of plots (5, 6, or 7)..... but it does not seem so.
Is it random and independent by the total amount of plots I have?
Is it normal? If so, why?
Thanks

**Comment by sonpython** [comment]
the passed filter ~= total plots / 512

50 plots => has 0, 1, 2 but 0 appears more usually than 1 2
244 plots => still 0, 1, 2, 3 but I think 1 and 2 will appear more usually than 0

**Comment by mkdev-code** [comment]
so its not farm just que for farm
